nginx/dev/adage-nginx.conf

```diff
@@ -0,0 +1,63 @@
+server {
+    # default server - drop requests without a Host header
+    listen 80 default_server;
+    server_name "";
+    return 444;
+}
+
+# configuration of the server
+server {
+    # the port your site will be served on
+    listen      80;
+    # the domain name it will serve for
+    server_name localhost;
+    charset     utf-8;
+    access_log on;
+
+    # max upload size
+    client_max_body_size 10M;   # adjust to taste
+
+    # /static is linked to static files required for use by the adage web application
+    location /static {
+        alias /home/static;
+    }
+
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
+    # / directs to the index.html in the static folder for the
+    # adage web application
+    # for detail on the P3P header, see http://www.w3.org/P3P/
+    location / {
+        root /home/static;
```

Comment:
Nice. Now I see how we bypass Django for serving up the interface. This combines with the `location /api` below to serve static interface files or proxy the REST API. I like it. It's good to have a clean separation here.